.github/workflows/build-orbit.yaml

```diff
@@ -14,6 +14,7 @@ on:
 
 env:
   ORBIT_VERSION: 1.17.0
+  CGO_ENABLED: 1
```

Comment:
What is this for?

Reply:
@getvictor IIRC it's for building the `app_icons` table which has a C dependency

Reply:
Yes, `app_icons` have C code that needs to be compiled in. It works on darwin and our CI, but I'm not sure if cross-compiling will work on Windows. @sharon-fdm do we need to support cross-compiling on Windows?